src/index.ts

```diff
@@ -134,6 +134,130 @@ app.post("/api/users", async (req, res) => {
   res.json(user);
 });
 
+/**
+ * @openapi
+ * /api/users/{id}:
+ *   patch:
+ *      summary: Update particular data for a user
+ *      description: Ur
+ *      parameters:
+ *        - name: id
+ *          in: path
+ *          description: User id to update
+ *          required: true
+ *          schema:
+ *            type: integer
+ *      requestBody:
+ *        description: Update customer with properties to be changed
+ *        content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             name: user data
+ *             required: true
+ *             properties:
+ *               name:
+ *                 type: string
+ *                 description: User name
+ *                 example: John Doe
+ *               email:
+ *                 type: string
+ *                 description: User email
+ *                 example: test@test.pl
+ *      responses:
+ *        200:
+ *          description: Successfully updated a user
+ *          content:
+ *            application/json:
+ *              schema:
+ *                type: object
+ *                properties:
+ *                  data:
+ *                      type: object
+ *                      properties:
+ *                        id:
+ *                          type: integer
+ *                          description: User id
+ *                          example: 1
+ *                        name:
+ *                          type: string
+ *                          description: Username
+ *                        email:
+ *                          type: string
+ *                          description: Username
+ *        400:
```

Comment:
Missing documentation for `404` response when user is missing

Reply:
fixed